backend/infrastructures/apps/extract/adapters/repositories.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+
+from django.core.files.base import ContentFile
+from django.core.files.storage import FileSystemStorage
+
+from modules.common.domain import exceptions as domain_exceptions
+from modules.extract.domain.ports import repositories as domain_repositories
+
+from ....config import settings
+
+
+class DjangoFileDomainRepository(domain_repositories.AbstractFileDomainRepository):
+    """
+    See description of parent class to get more details.
+    """
+
+    def save(self, file_bytes: bytes, file_name: str) -> Path:
```

Comment:
file: bytes


It's a bad practise to give a function type in variable name. For that we have type hints.